daily_milestone.go

```diff
@@ -156,15 +156,33 @@ func getMilestones(baseURL string, token string, projectID string) ([]simpleMile
 }
 
 // CreateMilestoneData creates new milestones with title and due date
-func createMilestoneData(advance int) []simpleMilestone {
+func createMilestoneData(advance int, milestoneTime string) []simpleMilestone {
 	today := time.Now().Local()
 	list := []simpleMilestone{}
 	for i := 0; i < advance; i++ {
-		date := today.AddDate(0, 0, i).Format("2006-01-02")
-		milestone := simpleMilestone{}
-		milestone.Title = date
-		milestone.DueDate = date
-		list = append(list, milestone)
+		if milestoneTime == "daily" {
```

Comment:
Long if else statements can sometimes be done using select statements: https://gobyexample.com/select

Reply:
Also the code is duplicated and might be moved outside the select:
```
milestone := simpleMilestone{}
milestone.Title = date
milestone.DueDate = date
list = append(list, milestone)
```

Reply:
Oh ok was thinking about avoiding duplicate code thanks